markdown/readme/languages/hindi.md

```
@@ -0,0 +1,162 @@
+# लाइसेंस
+
+<p>
+यह परियोजना <a href="./LICENSE">AGPLv3</a> के तहत लाइसेंस प्राप्त है।
+</p>
+
+# योगदानकर्ता
+
+आपके योगदान के लिए धन्यवाद! हम समुदाय से सभी योगदानों की सराहना करते हैं।
+
+<a href="https://github.com/nocodb/nocodb/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=nocodb/nocodb" />
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)